### ISSUE TITLE
Self-supervised Egomotion and Depth Learning via Bi-directional Coarse-to-Fine Scale Recovery. (arXiv:2211.08904v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23633]
Self-supervised Egomotion and Depth Learning via Bi-directional Coarse-to-Fine Scale Recovery. (arXiv:2211.08904v1 [cs.CV])<br>
https://ift.tt/bL4hvgc<br>
<p>Self-supervised learning of egomotion and depth has recently attracted great attentions. These learning models can provide pose and depth maps to support navigation and perception task for autonomous driving and robots, while they do not require high-precision ground-truth labels to train the networks. However, monocular vision based methods suffer from pose scale-ambiguity problem, so that can not generate physical meaningful trajectory, and thus their applications are limited in real-world. We propose a novel self-learning deep neural network framework that can learn to estimate egomotion and depths with absolute metric scale from monocular images. Coarse depth scale is recovered via comparing point cloud data against a pretrained model that ensures the consistency of photometric loss. The scale-ambiguity problem is solved by introducing a novel two-stages coarse-to-fine scale recovery strategy that jointly refines coarse poses and depths. Our model successfully produces pose and depth estimates in global scale-metric, even in low-light condition, i.e. driving at night. The evaluation on the public datasets demonstrates that our model outperforms both representative traditional and learning based VOs and VIOs, e.g. VINS-mono, ORB-SLAM, SC-Learner, and UnVIO.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
